tests/functional/object/mcg/test_namespace_crd.py

```diff
@@ -146,6 +146,23 @@ def deprecated_test_namespace_store_creation_crd(
                     pytest.mark.polarion_id("OCS-2407"),
                 ],
             ),
+            pytest.param(
+                {
+                    "interface": "CLI",
+                    "namespace_policy_dict": {
+                        "type": "Cache",
+                        "ttl": 300000,
+                        "namespacestore_dict": {
+                            "rgw": [(1, None)],
+                        },
+                    },
+                },
+                marks=[
+                    tier1,
+                    on_prem_platform_required,
+                    pytest.mark.polarion_id("OCS-2407"),
```

Comment:
Please create a new polarion item and reference it here instead of this one - this one doesn't match the test: 
```
OCS-2407 - Create MCG namespace bucket from RGW resource
```

Reply:
Done, thank you. Added also polarion test case and a proper mark for another test case related to one of my previous prs.